manuscript/03.results.md

```diff
@@ -62,7 +62,9 @@ Figure 4a depicts the similarity and difference between two states in terms of c
 [//]: # (Figure 6)
 
 This parts explains how the model performs in the case of an under-represented population of cells.
-The purpose of this section is to show the performance of the model when there are under-represented cell clusters in the population of interest. **To Do some introduction about the importance and existence of underrepresented clusters in solid tumors.** [@doi:10.1016/j.celrep.2019.10.045; doi:10.1371/journal.pcbi.1007278].
+The purpose of this section is to show the performance of the model when there are under-represented cell clusters in the population of interest. 
+Heterogeneous tumors have been exhibited to induce non-genetic resistance to anticancer drugs through exposure with other tumor cells. This previously unrecognized phenomenon indicates that it is necessary to consider underrepresented clusters in solid tumors, such as those that have gained resistance to anticancer drugs through environmental conditions, selective pressure, and cell-state transitions. In order to develop sufficient cancer therapy, it is important to recognize the existence of such tumor cells that gain resistance through non-genetic means and aren’t typically targeted for anticancer drugs so that scientists are able to accurately identify and predict the patterns that tumors gain drug resistance [@pmid: 31449515].Thus, it is relevant to consider both genetic and environmental drug resistance acquisition when determining tumor heterogeneity and its subsequent phenotypic variations in anticancer therapies. 
+[@doi:10.1016/j.celrep.2019.10.045; doi:10.1371/journal.pcbi.1007278].
```

Comment:
1. Your second sentence is a hard claim! You are claiming this phenomenon has never been recognized. Please don't use "never, always, ..." when writing a scientific report, paper, etc. because there might be a paper somewhere that you have not found, that has actually recognized this! Be careful!
2. It is not common to say for example, "... aren't ...", please use the full form "are not".
3. It is probably my mistake from before, but when you are citing, the second reference is missing "@". Please correct it to be "@doi:10.1371.....".
4. Try to put the reference at the end of a sentence that you have from that paper. A few first sentences are claims or results that should have references.